sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/util/CosmosPagedIterable.java

```diff
@@ -22,12 +22,27 @@
  */
 public final class CosmosPagedIterable<T> extends ContinuablePagedIterable<String, T, FeedResponse<T>> {
 
+    private CosmosPagedFlux<T> cosmosPagedFlux;
```

Comment:
nit: final.

Reply:
This cannot be final, because this is mutable. However, we can change it to be immutable, will try that out.